src/index.js

```diff
@@ -38,7 +38,7 @@ class AuthGoogleXoPlugin {
     xo.registerPassportStrategy(new Strategy(this._conf, async (accessToken, refreshToken, profile, done) => {
       try {
         console.log(profile)
-        done(null, await xo.registerUser('google', profile.displayName))
+        done(null, await xo.registerUser('google', profile.emails[0].value))
```

Comment:
Hmm, I wouldn't know how to do a fallback in JS. Passport.JS is being used here, correct?